youtube_dl_gui/optionsframe.py

```diff
@@ -633,6 +636,12 @@ def _set_sizer(self):
 
         vertical_sizer.AddSpacer(self.SIZE_10)
 
+        maxsimdls_sizer = wx.BoxSizer(wx.HORIZONTAL)
```

Comment:
You should add the maxsimdls_sizer to the same HORIZONTAL sizer with the retries_spintctrl this way you keep them side by side and it looks better.

Something like this
```python
top_hor_sizer = wx.BoxSizer(wx.HORIZONTAL)                              
                                                                        
top_hor_sizer.Add(self.retries_text)                                    
top_hor_sizer.AddSpacer(self.SIZE_5)                                    
top_hor_sizer.Add(self.retries_spinctrl)                                
                                                                                
top_hor_sizer.AddSpacer(self.SIZE_20)                                   
                                                                                
top_hor_sizer.Add(self.maxsimdls_text)                                  
top_hor_sizer.AddSpacer(self.SIZE_5)                                    
top_hor_sizer.Add(self.maxsimdls_spinctrl)                              
                                                                               
vertical_sizer.Add(top_hor_sizer)
```